### PR TITLE
ENH: Add 1D FFT Example

### LINKED ISSUE #354: Add 1D FFT examples
1D FFT filters were (relatively) recently ported from ITKUltrasound to ITK. Would be useful to have an example notebook demonstrating FFT plots (real, imaginary, modulus, phase) along one-dimensional scan lines. Meaningful example data can be taken from ITKUltrasound where one direction represents RF voltage waveform and other directions represent space or time dimensions.

### PR DESCRIPTION
**Pull request by tbirdso**
Takes in mouse liver RF image (public data from ITKUltrasound testing) and performs FFT along scanline.

Also adds option in `compare_to_baseline` CMake function to optionally allow only Python (no C++) baseline test creation.

Input

rf-input

Output modulus

MouseLiverModulusOutput

Output phase

MouseLiverPhaseOutput

Closes #354.

### CODE REVIEW

**Review by thewtex — APPROVED** [review]

**Comment by tbirdso** [comment]
Fixed reference to input screenshot in .rst documentation.

**Comment by tbirdso** [comment]
It looks like there is a larger issue of test dependencies here, the [log file](https://pipelines.actions.githubusercontent.com/serviceHosts/9bc39a5a-c4cc-43d6-ba38-df34c6a9a73c/_apis/pipelines/1/runs/1654/signedlogcontent/4?urlExpires=2022-05-23T16%3A06%3A07.2308814Z&urlSigningMethod=HMACV1&urlSignature=gi6XP04gJoPa2I%2F%2Fu3Nqx08es5bSrm4fN4xI8t4SHTs%3D) shows that `ComputeFFTInOneDimension` baseline comparisons are run before the `ComputeFFTInOneDimension` test. It appears that test dependencies are being set in CMake as one would expect. Need to dig deeper into this.

**Comment by tbirdso** [comment]
Note that it also looks like we will need to resolve [https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/issues/362](https://github.com/InsightSoftwareConsortium/ITKSphinxExamples/issues/362) before this one can be merged as 1D FFT classes were added after the 5.2.0 release.

**Pull request merged**
